src/lib.rs

```diff
@@ -37,6 +37,8 @@ pub enum Type {
     Rgb,
     /// FLIF (Free Lossless Image Format) files
     Flif,
+    /// ICO files
+    ICO
```

Comment:
Enum variants [should be](https://github.com/rust-lang/rfcs/blob/master/text/0430-finalizing-naming-conventions.md#general-naming-conventions) in UpperCamelCase, like an `Ico`